src/main/resources/org/jenkinsci/plugins/oic/OicServerManualConfiguration/help-jwksServerUrl.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Recommended. json webtoken key signature url of the openid connect provider
```

Comment:
original had `Recommended. jswon webtoken` which is fixed here and causing this not to show as a rename